Объективно лучшее обновление за историю Chaotic Onyx: Войны Хлорок

TobyThorne opened the pull request:
- Автоинъекторы теперь ОКОНЧАТЕЛЬНО И БЕСПОВОРОТНО АБСОЛЮТНО ТОЧНО НАВЕРНЯКА нельзя наполнять хлорочкой.
- Все автоинъекторы кроме оксикодонового и майндбрейкерного теперь содержат по 10 единиц вещества (5 единиц лекарств - это какое-то издевательство, в таблетках больше содержится).
- Добавлен инъектор с триком. Заменяет инапровалиновые инъекторы в красных коробках. Вроде как, они нужны чтобы спасать самого себя, а инапровалин нужен в основном для стабилизации критических пациентов.
- Убрано содержимое автоинъекторов из их названий. Теперь содержимое показывается в описании, вместе с короткой инструкцией по применению. Кроме того, все инъекторы и до этого имели разноцветные полоски, так что отличить можно без проблем (новый инъектор - с красной полоской).

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

Commit:
autoinjectors rework

- Autoinjectors are now COMPLETELY FINALLY ULTRA non-refillable
- All the autoinjectors (except oxycodone and mindbreaker) now hold and inject 10 units
- Added tricordrazine autoinjectors (replace inoprovaline autoinjectors in emergency boxes)
- Autoinjectors no longer have their contents in name, it's displayed in description along w/ short manual now

Shabat-Sholom commented:
Тоби, <3

Shabat-Sholom commented:
НАКОНЕЦ-ТО! Ес! ЕССС!

Commit:
Update hypospray.dm

TheUnknownOneGH commented:
> Автоинъекторы теперь ОКОНЧАТЕЛЬНО И БЕСПОВОРОТНО АБСОЛЮТНО ТОЧНО НАВЕРНЯКА нельзя наполнять хлорочкой.
> Все автоинъекторы кроме оксикодонового и майндбрейкерного теперь содержат по 10 единиц вещества (5 единиц лекарств - это какое-то издевательство, в таблетках больше содержится).

Уважение+

> Добавлен инъектор с триком. Заменяет инапровалиновые инъекторы в красных коробках. Вроде как, они нужны чтобы спасать самого себя, а инапровалин нужен в основном для стабилизации критических пациентов.

Ну если честно не совсем понимаю, зачем заменять инапровалиновые на трик. Мне всегда казалось, что набор из красных коробок нужен, чтобы продержаться в сознании чуть подольше, чтобы покинуть зону опасности и доковылять до медиков или же дожить до их прибытия. Инапровалин для этого хорошо подходит, но трик просто медленно лечит травмы, что тоже в какой-то степени помогает, но не настолько в ситуации, когда совсем жопа. Ну и к тому же врачам возможно придётся откачивать трик, который люди ранее сами себе вкололи, чтобы их оперировать.

TapKo4eB commented:
с трикопеном хуй его знает на беестешене вообще с эпинфрином шприцы который совмещает свойства иноправолина(буст стамины, вылечиванием обездвиживания, починкой лузбрефа) и при этом трика, но важно, что лечит дамаг только если тот меньше крит трешхолда

TapKo4eB reviewed (dismissed):
жить можно похуй

Commit:
Merge branch 'dev' of https://github.com/ChaoticOnyx/OnyxBay into finalinjectorfix

Commit:
poopdate

- Tweaked survival kits' textures;
- Added a security survival kit containing a tric autoinjector;

TobyThorne commented:
- Чутка переделал спрайты красных коробочек, теперь выглядят сексуальнее;
- Добавил security survival kit для СБшников, отличается от обычного наличием инъектора с триком и красного химсвета;
- Engineer survival kit теперь содержит оранжевый химсвет вместо зелёного;
- Инъектор с триком в обычном survival kit'е заменён на инъектор с иноправолином;

TapKo4eB reviewed (approved):
да
